EvidenceApi/StaffSelectedDocumentTypes.json

```diff
@@ -184,6 +189,26 @@
         "title": "NHS medical card",
         "description": ""
       },
+      {
+        "id": "immigration-document",
```

Comment:
Saw your comment about keeping this singular, as the user will select that for each document -- makes sense to me.